Kiosk/SettingsHelper.cs

```diff
@@ -339,6 +350,36 @@ private async void LoadRoamingSettings()
                 }
             }
 
+            value = ApplicationData.Current.RoamingSettings.Values["AutoRotateThroughDemos"];
+            if (value != null)
+            {
+                bool booleanValue;
+                if (bool.TryParse(value.ToString(), out booleanValue))
+                {
+                    this.AutoRotateThroughDemos = booleanValue;
+                }
+            }
+
+            value = ApplicationData.Current.RoamingSettings.Values["DemoRotationTimePerDemo"];
+            if (value != null)
+            {
+                int duration;
+                if (int.TryParse(value.ToString(), out duration))
+                {
+                    this.DemoRotationTimePerDemo = duration;
+                }
+            }
+
+            value = ApplicationData.Current.RoamingSettings.Values["ShowDialogOnFaceApiCallErrorsOnHowOldKiosk"];
+            if (value != null)
+            {
+                bool booleanValue;
+                if (bool.TryParse(value.ToString(), out booleanValue))
+                {
+                    this.ShowDialogOnFaceApiCallErrorsOnHowOldKiosk = booleanValue;
```

Comment:
Let's name this setting ShowDialogOnApiErrors instead (in the property name and also how it gets stored).

Reply:
Yes, this name is better. Fixed.